src/components/fields/TraceSelector.js

```diff
@@ -144,12 +145,28 @@ class TraceSelector extends Component {
       options: this.traceOptions,
       clearable: false,
     });
+    // Check and see if the advanced slector prop is true
+    const {advancedTraceTypeSelector} = this.context;
+    if (advancedTraceTypeSelector) {
+      return (
+        <div
+          className="trace-type-select-dropdown__wrapper"
+          onClick={() =>
+            this.context.openModal(<TraceTypeSelector {...props} />)
+          }
+        >
+          <UnconnectedDropdown {...props} />
```

Comment:
Just from a UI and visual standpoint, maybe it should look more like a button here, instead of an UnconnectedDropdown?

Reply:
Cool -- I will build out a button similar to what is in the workspace currently :)